src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectExtensibilityConfigurationFactory.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
```

Comment:
This entire class will eventually be re-constructed to support input from MSBuild and build the project extensibility pieces accordingly. For now the changes here are to just get the flow going and setup for success when we move to MSBuild.